gen/host.go

```diff
@@ -183,6 +183,145 @@ func genHost(g *protogen.GeneratedFile, f *fileInfo, service *serviceInfo) {
 		service.GoName,
 	))
 
+	// Open and LoadWithCompiled Functions
```

Comment:
@Ice3man543 , it's seem to applicable approach, but for common functionality `Load` can be combination of the `Open` and `LoadWithCompile`

Also cache option recommenced by @codefromthecrypt  can be validated to choose this approach or compilation cache.

Reply:
I've tried the cache approach and it's working fine. Closing!

Reply:
I recently came to more or less same issue, could you please share some links or docs on what is "cache approach" in context of go-plugin that you refer to?